sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ClientSideRequestStatistics.java

```diff
@@ -609,6 +619,9 @@ public void serialize(
             generator.writeObjectField("metadataDiagnosticsContext", statistics.getMetadataDiagnosticsContext());
             generator.writeObjectField("serializationDiagnosticsContext", statistics.getSerializationDiagnosticsContext());
             generator.writeObjectField("gatewayStatistics", statistics.gatewayStatistics);
+            if (statistics.samplingRateSnapshot < 1) {
```

Comment:
why only log it when it is less than 1. 
I get that it may never go above 1, but won't hurt to put it in diagnostics, may help in future to find a bug :)

Reply:
Always logging it now